core-schema/src/main/resources/sql-schema/V1__Create_initial_schema.sql

```diff
@@ -59,4 +59,39 @@ CREATE TABLE friends_friended_users (
   FOREIGN KEY (user_id) REFERENCES user_ids (id) ON DELETE CASCADE,
   FOREIGN KEY (friend_user_id) REFERENCES user_ids (id) ON DELETE CASCADE,
   PRIMARY KEY (user_id, friend_user_id)
+);
+
+CREATE TABLE clans_clan_info (
+  clan_id INT NOT NULL AUTO_INCREMENT UNIQUE,
+  clan_name VARCHAR(100) NOT NULL,
+  clan_leader INT NOT NULL,
+  clan_kills INT NOT NULL,
+  clan_deaths INT NOT NULL,
+  clan_assists INT NOT NULL,
+  PRIMARY KEY (clan_id)
+);
+
+CREATE TABLE clans_clan_membership (
+  user_id INT NOT NULL,
+  clan_id INT NOT NULL,
+  FOREIGN KEY (clan_id) REFERENCES clans_clan_info (clan_id) ON DELETE CASCADE,
+  FOREIGN KEY (user_id) REFERENCES user_ids (id) ON DELETE CASCADE,
+  PRIMARY KEY (user_id,clan_id)
+);
```

Comment:
```suggestion
  INDEX(clan_id)
);
```
I'd assume we're going to select against clan_id

Reply:
Good idea. I'll commit this alongside the rest of the changes.

Reply:
Fixed!